### DISCUSSION TITLE
AIR System requirements page required

### DISCUSSION THREAD

**Discussion by Tpman** [airsdk/Adobe-Runtime-Support#2200] [General]
Hi all.

Shouldn't the AIR System requirements page be available on Harman or AIR SDK sites?

The original link is as follows:
www.adobe.com/products/air/tech-specs.html

Thanks.

**Comment by marchbold** [comment]
Yep! :) 

Do we have a current tech specifications list we can publish?

**Reply by ajwfrost** [reply]
Yes this is one of those housekeeping issues that we generally haven't spent enough time on ... there are a few things that we need to take back still off Adobe i.e. where we currently just point people to Adobe's websites (which are often a bit more generic).

Best location I would say for system requirements - i.e. requirements of the computers that you can use the SDK on for developing AIR apps, and for then running AIR apps that you've built - are our release notes. But it would be good to consolidate these somewhere and have them also in a place where we can keep the history - for example, at some point in time we dropped support for 32-bit IPA generation, so it would be good to have a section that shows folk what SDK versions they'd need to download in order to still be able to do that, etc....

Probably we need to create a table and put this somewhere under the airsdk.dev site.

@Tpman  thanks for raising it!
